### PR TITLE
Feature-contact-form

### PR DESCRIPTION
**Pull request by jemutakiplagat**
-Add the contact form section on the footer.
-Add a background image on the form section
-Add mail contact on the form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,6 +189,51 @@ <h4 class="title">Skills</h4>
...
+              >
+                    <input type="text" class="text-form" id="name" name="username" required 
+                    maxlength="30" placeholder=" Fullname"><br>
+              
+                    <input type="Email" class="mail" id="email" name="email" required
+                     placeholder=" EmailAddress"><br>
+                
+                    <textarea class="subject" name="subject" required maxlength="500" placeholder=" Enter text here.." ></textarea>
+               
+                <button class="form-button" type="submit">Get In Touch</button>
```
Comment (HAMMAS-SALEEM) [inline index.html:211]: ### Your Button
student-form

### Figma Button
figma


- [x] Your button is not in the center as shown in the Figma template. Please make your button in center to follow the Figma template :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @jemutakiplagat :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitHub Flow

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions
**[Optional]**
- [ ] Your branch name is **feature** is not descriptive enough. You could have choose something like **feature/contact-section** which is more descriptive and shows your professionalism with your work :+1:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (HAMMAS-SALEEM) [inline .vscode/settings.json:1]: ### Your MIT file link
mit-link-fixable

### Expected MIT file link
fixed

- [ ] Please change your MIT file link in your readme as shown in the image above. This is the link to your license file of this project so please take it seriously :thinking:

Comment (HAMMAS-SALEEM) [inline .vscode/settings.json:1]: ### Your MIT.md file :spiral_notepad:
mitfile (copy)


- [x] Please add your name, current year, app type in your MIT.md file. This is the license file of your project so please take it seriously :thinking:

**Review by mugishasam123 — APPROVED** [review]
Hi @jemutakiplagat  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Good commit ✔️ 
- Descriptive README file 💯 

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**